Cargo.toml

```diff
@@ -149,25 +149,24 @@ tree_hash_derive = { git = "https://github.com/gear-tech/tree_hash.git", branch
 unroll = "0.1.5"
 
 # Gear/Substrate deps
-gstd = { version = "=1.5.0", features = ["nightly"] }
-gtest = "=1.5.0"
-gwasm-builder = { version = "=1.5.0", package = "gear-wasm-builder" }
-gmeta = "=1.5.0"
-gear-wasm-builder = { version = "=1.5.0", default-features = false }
-gsdk = "=1.5.0"
-gclient = "=1.5.0"
-gear-core = "=1.5.0"
-gear-wasm-instrument = "=1.5.0"
-gbuiltin-bls381 = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0" }
-gbuiltin-eth-bridge = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0" }
-pallet-gear-eth-bridge-rpc-runtime-api = { git = "https://github.com/gear-tech/gear.git", tag = "v1.5.0", default-features = false, features = [
+gstd = { version = "1.6.2", features = ["nightly"] }
+gtest = "1.6.2"
+gwasm-builder = { version = "1.6.2", package = "gear-wasm-builder" }
+gmeta = "1.6.2"
+gear-wasm-builder = { version = "1.6.2", default-features = false }
+gsdk = "1.6.2"
+gclient = "1.6.2"
+gear-core = "1.6.2"
+gbuiltin-bls381 = "1.6.1"
```

Comment:
why `1.6.1`?

Reply:
1.6.2 wasn't published and there are no changes between them